doc/schemas/fundchannel.request.json

```diff
@@ -40,6 +40,10 @@
       "items": {
         "type": "outpoint"
       }
+    },
+    "mindepth": {
+      "description": "Number of confirmations required before we consider the channel active",
+      "type": "u32"
     }
```

Comment:
THis implies we never tested this parameter (or at least, never as a named parameter!).

Reply:
The issue here is that the `fundchannel.request.json` file is new from this PR (the `fundchannel.schema.json` is the one that was tested).

When I backfilled it early in the PR we didn't have the `zeroconf` PR merged yet, hence why we needed to add it only later. And indeed CI and `make pytest` surfaced this lack of parameter in the first version of the JSON schema, hence why I added it here.

TL;DR: Things are working as expected, and we even verified that a missing param does trigger CI warnings.